src/common/components/eventSelector/EventSelector.tsx

```diff
@@ -47,9 +43,8 @@ const UmbrellaEventSelector: React.FC<UmbrellaEventSelectorProps> = ({
   const { data: eventsData, previousData: previousEventsData } = useEventsQuery(
```

Comment:
previousData is used to show previous options while fetching new events. Otherwise the event list would be cleared after typing anything to search input